src/server/frontend_xwayland/xwayland_surface.cpp

```diff
@@ -182,11 +182,11 @@ mf::XWaylandSurface::XWaylandSurface(
               {
                   is_transient_for(XCB_WINDOW_NONE);
               }),
-          property_handler<xcb_atom_t>(
+          property_handler<std::vector<xcb_atom_t>>(
               connection,
               window,
               connection->_NET_WM_WINDOW_TYPE,
-              [this](xcb_atom_t wm_type) { window_type(wm_type); },
+              [this](std::vector<xcb_atom_t> wm_types) { window_type(wm_types); },
```

Comment:
If we're going to pass a vector by value, then let's `move` it rather than copying.

Reply:
No reason to pass around by value at all